migration/README.md

```diff
@@ -0,0 +1,47 @@
+# Database Schema Migration
+
+For the database schema migration, we use the [Atlas tool](https://github.com/ariga/atlas).
+
+When you install the Bucketeer Helm application, the migration runs automatically before the Bucketeer application is installed.
+If the migration fails, it won't install the application.
+
+## Prerequisite
+
+Ensure that you have started the Minikube in the dev container and that the `localenv-mysql-0` pod is running without errors.
+
+## 1- Connecting to MySQL
+
+To connect to MySQL, you must enter the `localenv-mysql-0` pod and then connect to MySQL.
+
+```shell
+kubectl exec -it localenv-mysql-0 -- /bin/sh
+mysql -h localhost -u bucketeer -p bucketeer
```

Comment:
```suggestion
mysql -h localhost -u bucketeer -pbucketeer
```

> If you use a --password or -p option and specify a password value, there must be no space between --password= or -p and the password following it.

https://dev.mysql.com/doc/refman/8.4/en/connecting.html

Reply:
I'm specifying the database in the command, not the password.

Reply:
I see.
It would be nice to have the password or the link to it.

Reply:
There is a warning when executing the command not to do so for security reasons. I understand this is for local development, but getting that warning is annoying.

Reply:
It is not necessary to add the password to the command. But the password should be written somewhere else.

Reply:
The default [password](https://github.com/bucketeer-io/bucketeer/blob/main/manifests/bucketeer/values.dev.yaml#L33-L37) is `bucketeer`, but the developer configures it when setting up the local environment. So, it may be different depending on what the developer chooses.

I will write to add additional information.

Reply:
Done.